src/index.html

```diff
@@ -6,9 +6,42 @@
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500;700&display=swap" rel="stylesheet">
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header>
+      <a class="logo" href="index.html">
+        <img
+        class="logo_img"
+        src="images/logo.png"
+        alt="Logo MoYo"
+        >
+      </a>
+      <nav>
+        <ul class="nav_list">
+          <li class="nav_item"><a href="/1" class="is-active nav_link">Apple</a></li>
```

Comment:
Add empty lines between multiline sibling blocks of HTML

<img width="1188" alt="Screenshot 2023-12-05 at 21 29 30" src="https://github.com/mate-academy/layout_moyo-header/assets/64746943/701f0ee5-3490-40ea-924b-9945f1bec28f">